mava/components/jax/training/advantage_estimation.py

```diff
@@ -75,12 +79,15 @@ def gae_advantages(
             rewards = jnp.clip(rewards, -max_abs_reward, max_abs_reward)
 
             advantages = rlax.truncated_generalized_advantage_estimation(
-                rewards[:-1], discounts[:-1], self.config.gae_lambda, values
+                rewards[:-1],
+                discounts[:-1],
+                self.config.gae_lambda,
+                denormalize(stats, values),
```

Comment:
Minor thing. Instead of denormalising twice we can probably just denormalise once and pass it to both functions that need it. I am happy to benchmark it after this point :slightly_smiling_face:

Reply:
I fixed this.